examples/tools/autotools/autotools_toolchain/build_project_autotools_toolchain.rst

```diff
@@ -0,0 +1,214 @@
+
+First, we should install the requirements. For this example we didn't consider your profile, but ``fmt`` is higly compatible with many different configurations.
+
+.. code-block:: shell
+
+    conan install . --build=missing
+
+Now we are ready to build and run our **string_formatter** app:
+
+.. code-block:: shell
+
+    source conanbuild.sh
+
+The ``conanbuild.sh`` generated by the :ref:`VirtualBuildEnv<conan_tools_env_virtualbuildenv>` will fill all environment
+variables according to your profile, plus, Autotools, fmt .pc files and ``PKG_CONFIG_PATH`` as environment variable.
+
+.. code-block:: shell
+
+    aclocal
+    automake --add-missing
+    autoconf
+    ./configure
+    make
+
+Now we generate the ``configure`` and ``Makefile``, then we build our project by running the ``make`` command.
+
+As final result, we will have the binary ``string_formatter`` installed in our example folder.
+
+.. code-block:: shell
+
+    ./string_formatter
+    Conan - The C++ Package Manager!
+
+
+Building on Windows
+-------------------
+
+To build on Windows, we will use a pre-defined profile as an example to give a better view of our current configuration.
+Also, we will be running on the default command prompt, it can be invoked by ``cmd.exe``.
+
+.. code-block:: ini
+    :caption: **profiles/msys2**
+
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=gcc
+    compiler.exception=seh
+    compiler.libcxx=libstdc++
+    compiler.threads=posix
+    compiler.version=11
+    os=Windows
+    [options]
+    */*:shared=True
+    [conf]
+    tools.cmake.cmaketoolchain:generator=MSYS Makefiles
+    tools.microsoft.bash:subsystem=msys2
+    tools.microsoft.bash:active=True
+    [tool_requires]
+    msys2/cci.latest
+    autoconf/2.71
+    automake/1.16.5
+    pkgconf/1.9.3
+    [buildenv]
+    CC=gcc
+    CXX=g++
+
+With this profile, we will build on Windows, with MSYS2 support and GCC 11. Some build requirements will be needed too.
+
+As the first step, we will install MSYS2 Conan package, to be able to run bash inside of our command prompt.
+
+.. code-block:: batch
+
+    conan install -r conancenter --requires=msys2/cci.latest -g VirtualBuildEnv
+    conanbuild.bat
+    set MSYSTEM=MSYS
+    set MSYS2_PATH_TYPE=inherit
+    set CHERE_INVOKING=1
+    %MSYS_BIN%\bash.exe -l
+
+Once we have installed ``msys2/cci.latest`` package, we can activate the environment generated by :ref:`VirtualBuildEnv<conan_tools_env_virtualbuildenv>`.
+Then, we need to configure some environment variables required by MSYS2.
+
+The environment variable ``%MSYS_BIN%`` is configured by the script ``conanbuild.bat`` so we do not need to find where the Conan package is installed.
+As next step, we start the bash terminal embedded in the Conan package.
+
+This extras steps are only needed because we are building our project manually on the terminal,
+but in case it would be a regular Conan recipe, building a Conan package, it would not be necessary.
```

Comment:
what is `it` here? (both cases)

Reply:
Removed Windows section for simplification.